buildchain/buildchain/versions.py

```diff
@@ -305,36 +305,22 @@ def deb_full_name(self) -> str:
         PackageVersion(name='kubelet', version=K8S_VERSION),
         # Latest packages
         PackageVersion(name='containerd'),
-        PackageVersion(name='coreutils'),
         PackageVersion(name='cri-tools'),
-        PackageVersion(name='e2fsprogs'),
-        PackageVersion(name='ebtables'),
-        PackageVersion(name='ethtool'),
-        PackageVersion(name='genisoimage'),
-        PackageVersion(name='iproute'),
-        PackageVersion(name='iptables'),
```

Comment:
Do we really want to remove these packages from the listing? This means that, for example, we won't check if ISO tooling from `genisoimage` may not be available from the current repos, and our pre-checks wouldn't know. It also means that we can't enforce version constraints on some dependencies (e.g. `container-selinux`, which we know **must** be higher than 2.9x IIRC) if they are located in OS repositories.

I think this needs to be added back here, but it means handling them in buildchain differently, so let's move it to a future PR.

Reply:
We can add them back without changing how they're handled in the buildchain.
The script will just not download them if they're from base, updates or extras repositories.

Reply:
We need to change something in the buildchain as we don't want to fix the version (except maybe `contrainer-selinux` where we want a minimum version)